frontend/src/pages/Products/PageProductChild.vue

```diff
@@ -1,85 +1,124 @@
-		<q-header class="bg-white main_header">
-			<q-toolbar>
-				<div class="row">
-					<q-btn
-						flat
-						@click="drawer = !drawer"
-						round
-						color="primary"
-						icon="menu"
-					/>
-				</div>
-				<AccountSettings />
-			</q-toolbar>
-		</q-header>
-		<AjaxBar />
-
-		<q-drawer
-			side="right"
-			show-if-above
-			bordered
-			:width="250"
-			:breakpoint="500"
-			class="bg-grey-3"
-		>
-			<h6>Object Details</h6>
-			<div id="detailsContainer">
-				<ConfigPannel></ConfigPannel>
-			</div>
-		</q-drawer>
-		<Drawer v-model="drawer">
-			<template v-slot:drawerFilter v-if="isMounted">
-				<PaletteMenu id="paletteMenu" :data="paletteData"></PaletteMenu>
-			</template>
-			<template v-slot:drawerMenu>
-				<div class="q-pa-md q-gutter-sm" v-if="menu">
-					<q-tree
-						:nodes="menu"
-						node-key="label"
-						:filter="filter"
-						v-model:selected="selected"
-						default-expand-all
-					/>
-				</div>
+	<div>
+		<ProductHeader>
+			<template v-slot:productHeaderContent>
+				<section class="productDetails_navbar">
+					<div class="partOne">
+						<span class="text-bold">
+							<q-icon name="apps" size="20px" />
+							Product:
+						</span>
+						<span class="q-ml-sm">
+							{{
+								currentProductContent
+									? currentProductContent[0].name
+									: "Lodaing..."
+							}}
+						</span>
+					</div>
+					<div class="partTwo">
+						<q-btn-toggle
+							class="modelization_toggle q-ml-lg"
+							no-caps
+							rounded
+							elevated
+							toggle-color="primary"
+							color="white"
+							text-color="primary"
+							v-model="modelValue"
+							size="10px"
+							:options="[
+								{ label: 'Model 3D', value: vOne },
+								{ label: 'Text', value: vTwo },
+							]"
+						/>
+					</div>
+				</section>
 			</template>
-		</Drawer>
+		</ProductHeader>
+		<q-layout container style="height: 100vh" view="lHh lpR lFf">
+			<AjaxBar />
+
+			<!-- Left Drawer-->
+			<SideBar side="left" class="" v-if="modelValue === vOne">
+				<template v-slot:drawerContent>
+					<q-scroll-area class="fit">
+						<PaletteMenu id="paletteMenu" :data="paletteData"></PaletteMenu>
+					</q-scroll-area>
+				</template>
+			</SideBar>
 
-		<q-page-container class="q-pa-lg">
-			<q-page :style-fn="pageSizeTweak" class="q-mt-lg">
-				<ProductDetailsNavigation />
-			</q-page>
-		</q-page-container>
-	</q-layout>
+			<!-- Rigth Drawer-->
+			<SideBar side="right" class="" v-if="modelValue === vOne">
+				<template v-slot:drawerContent>
+					<h6 class="q-pl-md q-pt-md q-ma-none text-grey-8">Object Details</h6>
+					<div id="detailsContainer">
+						<ConfigPannel></ConfigPannel>
+					</div>
+				</template>
+			</SideBar>
+
+			<q-page-container class="q-pa-lg">
+				<q-page :style-fn="pageSizeTweak" class="q-mt-lg">
+					<Modelization v-if="modelValue === vOne" @getTreeObjects="tree" />
+					<Monaco v-show="modelValue === vTwo" />
```

Comment:
why don't you use v-else instead of v-show ?

Reply:
The v-show directive allows the monaco component to maintain its current state without data loss